tests/eclient/testdata/port_forward.txt

```diff
@@ -17,33 +17,80 @@ message 'Resetting of EVE'
 eden eve reset
 exec sleep 20
 
-message 'Creating networks'
-#exec sleep 5
+############################## TEST SCENARIO 1 ################################
+# Hairpin connectivity test between two apps connected to the same network instance
+
+message 'SCENARIO 1: Creating networks'
 eden network create 10.11.12.0/24 -n n1
-#exec sleep 5
-eden network create 10.11.13.0/24 -n n2
 
-test eden.network.test -test.v -timewait 10m ACTIVATED n1 n2
+test eden.network.test -test.v -timewait 10m ACTIVATED n1
 
-message 'Starting with both application attached to same network instance'
-eden pod deploy -v debug -n app1 docker://itmoeve/eclient:0.5 -p 2223:22 --networks=n1 --memory=512MB
-eden pod deploy -v debug -n app2 docker://itmoeve/eclient:0.5 -p 2224:22 --networks=n1 --memory=512MB
+message 'SCENARIO 1: Starting with both application attached to same network instance'
+eden pod deploy -v debug -n app1 docker://itmoeve/eclient:0.6 -p 2223:22 --networks=n1 --memory=512MB
+eden pod deploy -v debug -n app2 docker://itmoeve/eclient:0.6 -p 2224:22 --networks=n1 --memory=512MB
 
-message 'Waiting for apps to enter RUNNING state'
+message 'SCENARIO 1: Waiting for apps to enter RUNNING state'
 test eden.app.test -test.v -timewait 20m RUNNING app1 app2
 
-message 'Checking accessibility'
+message 'SCENARIO 1: Checking accessibility'
 exec -t 5m bash wait_ssh.sh 2223 2224
 
 eden eve status
 cp stdout eve_status
 
-message 'Testing port map connectivity between apps'
+message 'SCENARIO 1: Testing port map connectivity between apps'
+exec sleep 20
+exec -t 1m bash test_connectivity.sh 2223 2224
+stdout 'Ubuntu'
+
+message 'SCENARIO 1: Resource cleaning'
+eden pod delete app1
+eden pod delete app2
+
+test eden.app.test -test.v -timewait 10m - app1 app2
+
+eden network delete n1
+
+test eden.network.test -test.v -timewait 10m - n1
+
+stdout 'no network with n1 found'
+
+eden network ls
+! stdout '^n1\s'
+
+############################## TEST SCENARIO 2 ################################
+# Hairpin connectivity test between two apps connected to two different network instances.
+# These two network instances have different uplink adapters attached to them. And these adapters
+# get different IP addresses from the same subnet with DHCP.
+
+{{$osruntime := EdenOSRuntime "nan"}}
+
+{{if (eq $osruntime "linux")}}
```

Comment:
If I correctly understood, we rely here on network environment generated with qemu/VirualBox. But we have tests on GCP for now.
Can you please add here checks similar with checks in the workflow https://github.com/lf-edge/eden/blob/f2c9e5ed8937f80499e05afb8a08df44323c404d/tests/workflow/eden.workflow.tests.txt#L41

Reply:
Done. Pushed the changes.